core/experiments/assets/js/admin/behaviors/experiments-messages.js

```diff
@@ -25,19 +25,26 @@ export default class ExperimentsDependency {
 		} );
 	}
 
+	inactiveStateActivity( experimentId ) {
+		if ( !! this.isExperimentContainsDeactivatingMessage( experimentId ) && elementorAdminConfig.experiments[ experimentId ].state !== STATE_INACTIVE ) {
```

Comment:
extract the global variable access to local named variable (e.g. `initialState`, `originalState`, etc.)

Reply:
Function removed